src/App.tsx

```diff
@@ -1,24 +1,220 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import {
+  deleteTodo,
+  getTodos,
+  patchTodo,
+  postTodo,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { TodosList } from './components/TodosList/TodosList';
 
-const USER_ID = 0;
+const USER_ID = 6937;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasError, setHasError] = useState('');
+  const [query, setQuery] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo>();
+  const [isDisabledInput, setIsDisabledInput] = useState(false);
+  const [completedId, setCompletedId] = useState<number[]>([]);
+  const [isDeleting, setIsDeleting] = useState(false);
+
+  const waitErrorEnable = () => {
+    setTimeout(() => {
+      setHasError('');
+    }, 3000);
+  };
+
+  const addTodo = (title: string) => {
+    setIsDisabledInput(true);
+
+    const newTodo = {
+      title,
+      userId: USER_ID,
+      completed: false,
+    };
+
+    setTempTodo({
+      ...newTodo,
+      id: 0,
+    });
+
+    postTodo(newTodo)
+      .then(data => {
+        setTodos(prevState => [...prevState, data]);
+      })
+      .catch(() => {
+        setHasError('Unable to add a todo');
+        waitErrorEnable();
+      })
+      .finally(() => {
+        setTempTodo(undefined);
+        setIsDisabledInput(false);
+      });
+  };
+
+  const removeTodo = (id: number) => {
+    setCompletedId(state => [...state, id]);
+
+    return deleteTodo(id)
+      .then(() => {
+        setTodos(todos.filter(todo => todo.id !== id));
+      })
+      .catch(() => {
+        setHasError('Unable to delete a todo');
```

Comment:
Better to save error messages to constants, or even to enum
That way will help you avoid magic strings, and help to manage messages in one place